spec/javascripts/packages/document-capture/components/acuant-sdk-spec.js

```diff
@@ -2,50 +2,57 @@
  * Acuant SDK Loading Tests
  *
  */
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
 import { JSDOM } from 'jsdom';
-import AcuantJavascriptWebSdk from '../../../../../public/acuant/11.8.1/AcuantJavascriptWebSdk.min.js';
-
-const sdkPaths = {
-  '11.8.1': '../../../../../public/acuant/11.8.1/AcuantJavascriptWebSdk.min.js',
-};
-
-const TEST_URL = `file://${__dirname}/index.html`;
-
-const { window } = new JSDOM(
-  '<!doctype html><html lang="en"><head><title>JSDOM</title></head></html>',
-  {
-    url: TEST_URL,
-    runScripts: 'dangerously',
-    resources: 'usable',
-  },
-);
-const { document } = window;
-
-describe('Acuant SDK Loading Tests', () => {
-  it('Can load something from the SDK file', () => {
-    expect(AcuantJavascriptWebSdk).to.exist();
-  });
-  describe('DOM Loading 11.8.1', () => {
-    before((done) => {
-      const scriptEl = document.createElement('script');
-      scriptEl.id = 'test-acuant-sdk-script';
-      scriptEl.onload = () => {
-        done();
-      };
-      scriptEl.src = sdkPaths['11.8.1'];
-      document.body.append(scriptEl);
-    });
-    it('There is a script element in the DOM', () => {
-      const found = document.getElementById('test-acuant-sdk-script');
-      expect(found).to.exist();
-    });
-    it('Has a global loadAcuantSdk object on the window', () => {
-      expect(window.loadAcuantSdk).to.exist();
-    });
-    it('Calling loadAcuantSdk gives us AcuantJavascriptWebSdk in the global scope and as a prop of the window', () => {
-      window.loadAcuantSdk();
-      expect(AcuantJavascriptWebSdk).to.exist();
-      expect(window.AcuantJavascriptWebSdk).to.exist();
+
+const ACUANT_PUBLIC_DIR = '../../../../../public/acuant';
+const VERSION_REGEX = /^\d+\.\d+\.\d+$/;
+
+describe('Acuant SDK Loading Tests', async () => {
+  const sdks = (await fs.readdir(path.join(__dirname, ACUANT_PUBLIC_DIR))).filter((dir) =>
+    VERSION_REGEX.test(dir),
+  );
```

Comment:
Is there some way we could ensure that this is not an empty array, such as if the folder path or naming convention changes? (Similar idea to [Jest's Assertion Count assertions](https://jestjs.io/docs/expect#assertion-count))


Wondering if something like this could work:

```suggestion
  const sdks = (await fs.readdir(path.join(__dirname, ACUANT_PUBLIC_DIR))).filter((dir) =>
    VERSION_REGEX.test(dir),
  );

  if (!sdks.length) {
    throw new Error('Expected to find at least one SDK version, but found none');
  }
```